rules/Configs/Rector/ClassMethod/AddRouteAnnotationRector.php

```diff
@@ -84,6 +86,7 @@ public function refactor(Node $node): ?Node
             }
 
             $hasChanged = true;
+            $this->docBlockUpdater->updateRefactoredNodeWithPhpDocInfo($classMethod);
```

Comment:
@TomasVotruba this is the reason the docblock was not updated, this patch fix it.

Reply:
Thank you 🙏